### ISSUE TITLE
Feature request: +1/+1 bonus counter

### ISSUE THREAD

**Issue by Capitoste** [brine/mtg-octgn#86]
It would be good to have a new +1+1 bonus counter, different from that blue +1+1 counter already available. So often we have to differenciate between actual +1+1 counters (the ones that remain in permanents) from +1+1 bonuses (the ones that are given from cards like honor of the pure or elvish archdruid).

**Comment by DotChris** [comment]
While I'm sure adding a new marker type wouldn't be too difficult of a request,(at a glance looks like)

```
def addPlusOneBonus(card, x = 0, y = 0):

    mute()
    notify("{} adds a +1/+1 bonus to {}.".format(me, card))
   [logic here]

```
(obviously there is some other stuff like graphics and adding it in other places too)

 I wondered if there was a way to script it instead. I tried using the scryfall api to target cards that add +x/+x using the regex `re:"( get | gets )(+\d/+\d)"` or` https://api.scryfall.com/cards/search?q=re%3A%22(+get+|+gets+)(%2B\d%2F%2B\d)%22` to grab the list of cards. One could then cache this data and then narrow down the list further by setting cases for "other" "this" "target" etc.  I haven't looked too much into if power/toughness are static or loaded onto cards dynamically, but this would be a cool option and less time keeping track of bonuses.

**Comment by brine** [comment]
I've been avoiding power/toughness modifiers mainly as I haven't figured out a proper way to display them, I really don't like the idea of using counters as it gets way too messy when there's different counters on a card (especially +1/+1 counters).  We don't really have a way to put text overlays over the cards, which would be IMO the cleanest way to handle it.

**Comment by DotChris** [comment]
I agree, maybe for now, it would be a step to at least calculate bonuses and counters. Then, maybe a button that says calculate power/toughness and outputs it to chat for everyone. This would definitely be some major updating.

On the visual note: 
Would have to use a blank P/T area like in the newer cards probably white because it'll be easier to modify to other colors.
Select Color(maybe prefab each version)
Calculate P/T
Then place on (X,Y) of the card.

I have no idea on how to implement the images with OCTGN, maybe some derivative of markers. Pretty new to OCTGN's stuff, seeing that the scripts are in python is it possible to implement pillow to add images and text?

**Comment by brine** [comment]
well one thing I've been pushing for is the ability to have proxy images render whenever a card property changes (as python allows you to change the default values of a card property), which would in theory allow cards to display modified values
